src/api/flaskr/service/shifu/dtos.py

```
@@ -437,20 +447,23 @@ def __json__(self):
 @register_schema_to_swagger
 class TextInputDto(InputDto):
     # type text input
+    profile_ids: list[str]
     prompt: AIDto
     input_name: str
     input_key: str
     input_placeholder: str
 
     def __init__(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)